src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -4840,6 +4850,16 @@ instruct loadConF(regF dst, immF con) %{
   ins_pipe(pipe_slow);
 %}
 
+instruct loadConH(regF dst, immH con) %{
+  match(Set dst con);
+  ins_cost(125);
+  format %{ "movss   $dst, [$constantaddress]\t# load from constant table: halffloat=$con" %}
+  ins_encode %{
+    __ movflt($dst$$XMMRegister, $constantaddress($con));
```

Comment:
Are we storing a 32 bit value in ConstantTable for half float? Just checking that this is not introducing reading beyond the memory?